src/features/Admin/Ads/Ads.tsx

```diff
@@ -1,9 +1,475 @@
-    </div>
+    <>
+      <AppBar position="static">
+        <div className={style.header}>
+          <Typography variant="h6">
+            Rooms Table Details
+            <p variant="h6">You can check all details</p>
+          </Typography>
+
+          <Link to="/home/ads/add-ad">
+            <CustomButton
+              className="your-custom-class"
+              type="submit"
+              fullWidth
+              variant="contained"
+              sx={{ mt: 3, mb: 2 }}
+            >
+              Add new Ad
+            </CustomButton>
+          </Link>
+        </div>
+      </AppBar>
+      <div style={{ marginTop: '40px' }}></div>
+      <div>
+        {/* Table */}
+        <TableContainer component={Paper}>
+          <Table>
+            <TableHead>
+              <TableRow>
+
+                <TableCell>Room Number</TableCell>
+                <TableCell>Price</TableCell>
+                <TableCell>Discount</TableCell>
+                <TableCell>Capacity</TableCell>
+                <TableCell>Active</TableCell>
+                <TableCell>Action</TableCell>
+                {/* <TableCell align="center" valign="middle">Actios</TableCell> */}
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {adsList?.length > 0 &&
+                adsList.map((ad) => (
+                  <TableRow key={ad?._id}>
+                    <TableCell align="center" valign="middle">{ad?.room?.roomNumber}</TableCell>
+                    <TableCell align="center" valign="middle">{ad?.room?.price}</TableCell>
+                    <TableCell align="center" valign="middle">{ad?.room?.discount}</TableCell>
+                    <TableCell align="center" valign="middle">{ad?.room?.capacity}</TableCell>
+                    <TableCell align="center" valign="middle">{ad?.isActive ? 'Yes' : 'No'}</TableCell>
+                    <TableCell>
+                      <IconButton onClick={(e) => handleMenuClick(e, ad)}>
+                        <MoreVertIcon />
+                      </IconButton>
+                      <Menu
+                        anchorEl={anchorEl}
+                        // open={Boolean(anchorEl)}
+                        open={Boolean(anchorEl && selectedAd?._id === ad?._id)}
+                        onClose={handleClose}
+                      >
+                        <MenuItem
+                          onClick={() => showViewModal(ad?._id)}
+                        >
+                          <Tooltip title="View" arrow>
+                            <IconButton color="primary" >
+                              <VisibilityIcon fontSize='small' />
+                              <p style={{ fontSize: '14px' }}>View</p>
+                            </IconButton>
+                          </Tooltip>
+                        </MenuItem>
+                        <MenuItem
+                          onClick={() => showUpdateModal(ad)}>
+                          <Tooltip title="Update" arrow>
```

Comment:
Tooltip component is responsible for showing a tip about the button that we are using when hovering.
If you hover over any menu item, it will show you the title passed to the tooltip.
This is beneficial when we only have icons, but in our case,  we are explicitly writing what the icons are doing, so we can consider removing the tooltip around the menu items.